### ISSUE TITLE
`names` and `paths` not aligned in ImageList class

### ISSUE THREAD

**Issue by silvbarb** [fmicompbio/ez_zarr#12]
In `ez_zarr 0.2.1` paths and names are wrongly assigned within the image class when using `import_Fractal_plate`

Example: 
```
plate = ome_zarr.import_Fractal_plate(zarr_directory) # zarr_directory can be any Fractal-test plates
im = plate['B02'] # pick one well to showcase
im.get_paths() # returns .../_mip.zarr/G/02/0' # requested well B02, but returned image at 'G02'
```

**Comment by csoneson** [comment]
Thanks for the report - indeed I think there is an unmatched sorting step, sorry about that. 
Could you try to install `ez-zarr` from the `fix-plate-sort` branch and see if that does it on your data set for now? We will fix this in the release version once back from holidays.

**Comment by silvbarb** [comment]
I installed the `fix-plate-sort` branch which seems to work on my data set, now: 

```
plate = ome_zarr.import_Fractal_plate(zarr_directory) # zarr_directory can be any Fractal-test plates
im = plate['B02'] # picking one well ... 
im.get_paths() # returns the expected path .../_mip.zarr/B/02/0' # requested well B02, and returned image at 'G02'
```

Thanks!

**Comment by csoneson** [comment]
Great, thanks for checking! I have added some more unit tests as well 🙂

**Comment by mbstadler** [comment]
This has now been released on GitHub and PyPi (version 0.2.3, https://pypi.org/project/ez-zarr/).
